server/pulp/plugins/util/misc.py

```diff
@@ -142,3 +143,89 @@ def clear_directory(path, skip_list=()):
 
         elif os.path.isfile(entry_path):
             os.unlink(entry_path)
+
+
+def common_path(dirs):
+    _common_path = os.path.commonprefix(dirs)
+    if not _common_path.endswith("/"):
+        _common_path = os.path.dirname(_common_path)
+    return _common_path
+
+
+def make_link_unit(unit, extra_dst_path, working_dir,
```

Comment:
This function and the next two look like they are probably specific to the rsync distributor. I think there is probably a better place to put them than a ``util.misc`` module.